MonoGame.Framework/Graphics/Texture2D.DirectX.cs

```diff
@@ -86,76 +88,79 @@ private void PlatformConstruct(int width, int height, bool mipmap, SurfaceFormat
             var levelWidth = Math.Max(width >> level, min);
             var levelHeight = Math.Max(height >> level, min);
 
-            var desc = new Texture2DDescription();
-            desc.Width = levelWidth;
-            desc.Height = levelHeight;
-            desc.MipLevels = 1;
-            desc.ArraySize = 1;
-            desc.Format = SharpDXHelper.ToFormat(_format);
-            desc.BindFlags = BindFlags.None;
-            desc.CpuAccessFlags = CpuAccessFlags.Read;
-            desc.SampleDescription.Count = 1;
-            desc.SampleDescription.Quality = 0;
-            desc.Usage = ResourceUsage.Staging;
-            desc.OptionFlags = ResourceOptionFlags.None;
-
-            // Save sampling description.
-            _sampleDescription = desc.SampleDescription;
+            if (cachedStagingTexture == null)
+            {
+                var desc = new Texture2DDescription();
+                desc.Width = levelWidth;
+                desc.Height = levelHeight;
+                desc.MipLevels = 1;
+                desc.ArraySize = 1;
+                desc.Format = SharpDXHelper.ToFormat(_format);
+                desc.BindFlags = BindFlags.None;
+                desc.CpuAccessFlags = CpuAccessFlags.Read;
+                desc.SampleDescription.Count = 1;
+                desc.SampleDescription.Quality = 0;
+                desc.Usage = ResourceUsage.Staging;
+                desc.OptionFlags = ResourceOptionFlags.None;
+
+                // Save sampling description.
+                _sampleDescription = desc.SampleDescription;
+
+                cachedStagingTexture = new SharpDX.Direct3D11.Texture2D(GraphicsDevice._d3dDevice, desc);
```

Comment:
The `cachedStagingTexture` needs to be disposed when the `Texture2D` is disposed.  You will need to override `Dispose` from `Texture`...

https://github.com/MonoGame/MonoGame/blob/develop/MonoGame.Framework/Graphics/Texture.DirectX.cs#L52